src/simularium/VisGeometry/GeometryStore.ts

```diff
@@ -0,0 +1,352 @@
+import { forEach } from "lodash";
+import { OBJLoader } from "three/examples/jsm/loaders/OBJLoader";
+import jsLogger, { ILogger, ILogLevel } from "js-logger";
+import { BufferGeometry, Object3D, Mesh } from "three";
+
+import { checkAndSanitizePath } from "../../util";
+import PDBModel from "../PDBModel";
+import { InstancedMesh } from "../rendering/InstancedMesh";
+import VisAgent from "../VisAgent";
+import TaskQueue from "../worker/TaskQueue";
+import { AgentTypeVisData } from "../types";
+
+import {
+    AgentGeometry,
+    GeometryDisplayType,
+    GeometryStoreLoadResponse,
+    MeshGeometry,
+    MeshLoadRequest,
+} from "./types";
+
+export const DEFAULT_MESH_NAME = "SPHERE";
+
+type Registry = Map<string, AgentGeometry>;
+
+class GeometryStore {
```

Comment:
Can we have documented somewhere (in a readme, or as code comments for class definitions) the intended purpose/scope of `GeometryStore` and `VisGeometry`? That would be sooo helpful. (I would love them for `VisAgent` and `VisData` too but I don't mean in this PR 🙂)

Reply:
I added some comments to the file itself. Maybe the planned file reorg can also add a README to this folder?

Reply:
That sounds good. I don't see it in GeometryStore.ts though, did you push it?

Reply:
Now I did. :)

Reply:
Beautiful thank you 😍